Agar.io/Assets/Scripts/Model/Entity.cs

```diff
@@ -5,8 +5,9 @@ namespace Agario.Model
     public abstract class Entity
     {
         public Vector2 Position { get; set; }
-        public float Radius { get; set; }
+        public int Radius { get; set; }
```

Comment:
Why it's now an int?

Reply:
For udp packets it is better to use small size types. I thought that int was smaller than float. But i checked and they have the same size. I return float